public/pptc/case_lists/cases_all.txt

```diff
@@ -1,5 +1,5 @@
-cancer_study_identifier: pptc_2018
-stable_id: pptc_2018_all
+cancer_study_identifier: mixed_pptc_2018
```

Comment:
@ritikakundra I think for mixed studies, we should not use mixed in the ID, e.g. msk_impact_2017. I think pptc_2018 is fine.